README.md

```diff
@@ -73,16 +75,35 @@ as otherwise the build will perform `npm install` which will overwrite the links
 After you successfully execute `npm run tests` open the Gameface player or Chrome with "--url=http://localhost:9876/debug.html" to see the tests running.
 
 
-Creating new Components
-===================
+# Building from source
+
+To build the components from this repository use the `npm run build:dev` command. This will: 
+1. create symlinks for all components
+2. build all components and their demos
+NPM will use the links to install the dependencies, not the public npm packages. This means that if a component depends on 
+another, the dependency will be installed from the source, making it easy to test local changes.
+For example - the dropdown component depends on the scrollable container. If you make changes to the scrollable-container and
+run `npm run build:dev` you'll be able to observe the changes that you did to the scrollable-container in
+the demo of dropdown, because it uses the local package created by the build:dev command. If you inspect the scrollable-container package located in the dropdown's node_modules you'll see that it is a symbolic link that references the source of the scrollable container located in the components folder.
+
+The `npm run build:dev` command will build all components. If you are not changing all of them you don't need to
+rebuild them every time. You can build individual components using the coherent game UI components CLI.

```

Comment:
Can we create a new section on how to build components with the CLI?

Reply:
It is explained om the CLI docs. We don't want to repeat the content from the CLI documentation, this is why we add link to it, instead of explaining it here as well.